modules/hash_demulti/bff.nf

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+process bff{
+    publishDir "$projectDir/$params.outdir/$params.mode/hash_demulti/bff", mode:'copy'
+    input:
+        each seurat_object
+        //Seurat process
+        each assay
+        each methods
+        each methodsForConsensus
+        each cellbarcodeWhitelist
+        each metricsFile
+        each doTSNE
+        each doHeatmap
+        each perCellSaturation
+        each majorityConsensusThreshold
+        each chemistry
+        each callerDisagreementThreshold
+        each assignmentOutBff
+        
+    output:
+        path "bff_${task.index}"
+        
+    script:
+        def generateGenderPlot = generate_gender_plot != 'None' ? " --generateGenderPlot ${generate_gender_plot}" : ''
```

Comment:
why do we need this?

Reply:
The process failed because of this line, please be really sure that the process works before making pull request :)